src/inventory.gql

```diff
@@ -116,6 +140,12 @@ type Mutation {
     """UUID of product item to delete"""
     id: UUID!
   ): ProductItem!
+
+  """Reserves a product item of a chosen product variant"""
+  reserveProductItem(
+    """UUID of product variant to reserve"""
+    productVariant: UUID!
```

Comment:
maybe it would make sense to create an input object and also allow a count of how many to reserve?
because shoppingcart items also have a count, if we want to do an atomic mapping here that would be helpful

Reply:
isnt this possible over the products connection?

Reply:
how does the connection help you for batch creating?

Reply:
I wil move this to a seperate issue